tests/unittest/llmapi/apps/_test_openai_embeddings.py

```
@@ -0,0 +1,385 @@
+
+def test_pretokenized_input(client: openai.OpenAI):
+    # A pre-tokenized single input (list[int]) must be accepted as-is.
+    token_ids = [101, 7592, 2088, 102]  # [CLS] hello world [SEP]
+    response = client.embeddings.create(model=BERT_MODEL, input=token_ids)
+    assert len(response.data) == 1
+    assert len(response.data[0].embedding) == NUM_CLASSES
+
+
+def test_base64_encoding_format(server: RemoteEmbeddingServer):
+    # Use raw requests to assert the wire format precisely.
+    resp = requests.post(
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)